kernel/kernel.mld

```diff
@@ -0,0 +1,6 @@
+{1 The Coq kernel}
+
+This chapter describes the **Coq** kernel, which is a type checker for
+the Calculus of Inductive Constructions.
```

Comment:
I didn't see this anywhere in the odoc rendered HTML.

Reply:
This is placed on the `coq/kernel.html` file, but it is not linked from anywhere. I think we will have to wait for `odoc` 1.3.0 [or maybe it will be called 2.0] to see what we do; there are many new features being added so for now really this PR has mostly "witness" status so we can see how odoc does and we are sure docs build with it.